src/mcpdotnet/Server/RequestContext.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace McpDotNet.Server;
+
+/// <summary>
+/// Container for the request context.
+/// </summary>
+/// <typeparam name="TParams">Type of the request parameters</typeparam>
+public record RequestContext<TParams>(IMcpServer Server, TParams? Params)
+{
+
+}
```

Comment:
Nit: this can be expressed with just a semicolon rather than braces:
```C#
public record RequestContext<TParams>(IMcpServer Server, TParams? Params);
```